specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt

```diff
@@ -15,7 +15,7 @@
 #
 
 property writeMask ${httpx:randomBytes(4)}
-connect http://localhost:8001/echo

```

Comment:
It might be best to use 8080 and "path" like the other scripts.  Just a suggestion though